src/Compilers/Core/Portable/Operations/OperationWalker.cs

```diff
@@ -40,454 +37,20 @@ public override void Visit(IOperation operation)
             }
         }
 
-        internal override void VisitNoneOperation(IOperation operation)
+        public override void DefaultVisit(IOperation operation)
         {
             if (operation is IOperationWithChildren operationWithChildren)
             {
                 VisitArray(operationWithChildren.Children);
             }
         }
 
-        public override void VisitBlockStatement(IBlockStatement operation)
```

Comment:
I thought this entry was marking a override of a public method: https://github.com/dotnet/roslyn/blob/features/ioperation/src/Compilers/Core/Portable/PublicAPI.Unshipped.txt#L745 and would need to be deleted with your change.